core/web/schema/type/spec.graphql

```diff
@@ -89,6 +89,7 @@ type OCR2Spec {
     transmitterID: String
     pluginType: String!
     pluginConfig: Map!
+    feedID: String!
```

Comment:
This has a `!` indicating it will never be null, but feedID is not always present... does it come as an empty string if its missing?